src/screens/CreatePost/index.js

```diff
@@ -936,7 +935,7 @@ const styles = StyleSheet.create({
     height
   }),
   reminderContainer: {
-    backgroundColor: '#2F80ED',
+    backgroundColor: colors.blue,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hard-coded color value has been replaced with a reference to the `colors` object. This is a good practice as it allows for easier management and updating of color schemes.
```diff
- backgroundColor: '#2F80ED',
+ backgroundColor: colors.blue,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->